.env.example

```
@@ -47,6 +47,11 @@ LANGFUSE_METADATA_MIRROR_SECRET_KEY=
 ##############################################################################
 CHAT_CONTEXT_WEB_SEARCH_BRAVE_API_KEY=
 CHAT_CONTEXT_WEB_SEARCH_SERPAPI_API_KEY=
+# GitHub context runtime controls: CHAT_CONTEXT_GITHUB_ENABLED,
+# CHAT_CONTEXT_GITHUB_TIMEOUT_MS, CHAT_CONTEXT_GITHUB_MAX_RECORDS_PER_SECTION,
+# CHAT_CONTEXT_GITHUB_PRIVATE_REPOSITORY_ALLOWLIST, CHAT_CONTEXT_GITHUB_CACHE_TTL_MS,
+# and CHAT_CONTEXT_GITHUB_STALE_RESULT_LIMIT_MS. Defaults and descriptions live in config-spec.
+CHAT_CONTEXT_GITHUB_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHAT_CONTEXT_GITHUB_TOKEN key should go before the CHAT_CONTEXT_WEB_SEARCH_BRAVE_API_KEY key

(UnorderedKey)